docs/v2/spec/2018_Modified_LAR_Data_Dictionary.csv

```diff
@@ -0,0 +1,86 @@
+Race of Co-Applicant or Co-Borrower: 4,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Co-Applicant or Co-Borrower: 5,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3
+Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3 4
+Sex of Applicant or Borrower,Numeric,1 2 3 4 6
+Sex of Co-Applicant or Co-Borrower,Numeric,1 2 3 4 5 6
+Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3
+Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3 4
+Age of Applicant or Borrower,Character Varying,<25 25-34 35-44 45-54 55-64 65-74 >74
+Age of Applicant >= 62,Character Varying,Yes; No; NA
+Age of Co-Applicant or Co-Borrower,Character Varying,<25; 25-34; 35-44; 45-54; 55-64; 65-74; >74
+Age of Co-Applicant >= 62,Character Varying,Yes; No; NA
+Income,Character Varying, 'NA'
+Type of Purchaser,Numeric,0 1 2 3 4 5 6 71 72 8 9
+Rate Spread,Character Varying,'NA' 'Exempt'
+HOEPA Status,Numeric,1 2 3
+Lien Status,Numeric,1 2
+Applicant or Borrower - Name and Version of Credit Scoring Model,Numeric,1 2 3 4 5 6 7 8 9 1111
+Co-Applicant or Co-Borrower - Name and Version of Credit Scoring Model,Numeric,1 2 3 4 5 6 7 8 9 10 1111
+Reason for Denial: 1,Numeric,1 2 3 4 5 6 7 8 9 10 1111
+Reason for Denial: 2,Character Varying,1 2 3 4 5 6 7 8 9 blank
+Reason for Denial: 3,Character Varying,1 2 3 4 5 6 7 8 9 blank
+Reason for Denial: 4,Character Varying,1 2 3 4 5 6 7 8 9 blank
+Total Loan Costs,Character Varying, 'NA' 'Exempt'
+Total Points and Fees,Character Varying, 'NA' 'Exempt'
+Origination Charges,Character Varying, 'NA' 'Exempt'
+Discount Points,Character Varying, 'NA' 'Exempt'
+Lender Credits,Character Varying, 'NA' 'Exempt'
+Interest Rate,Character Varying, 'NA' 'Exempt'
+Prepayment Penalty Term,Character Varying, 'NA' 'Exempt'
+Debt-to-Income Ratio,Character Varying,<20% 20%- <30% 30%-<36% 50%-60% >60% Exact values between 36%-<50% NA Exempt
+Combined Loan-to-Value Ratio,Character Varying, 'NA' 'Exempt'
+Loan Term,Character Varying, 'NA' 'Exempt'
+Introductory Rate Period,Character Varying, 'NA' 'Exempt'
+Balloon Payment,Numeric,1 2 1111
+Interest-Only Payments,Numeric,1 2 1111
+Negative Amortization,Numeric,1 2 1111
+Other Non-amortizing Features,Numeric,1 2 1111
+Property Value,Numeric, 'NA' 'Exempt'
+Manufactured Home Secured Property Type,Numeric,1 2 3 1111
+Manufactured Home Land Property Interest,Numeric,1 2 3 4 5 1111
+Total Units,Character Varying,1 2 3 4 5-24 25-49 50-99 100-149 >149
+Multifamily Affordable Units,Character Varying, 'NA' 'Exempt'
+Submission of Application,Numeric,1 2 3 1111
+Initially Payable to Your Institution,Numeric,1 2 3 1111
+Automated Underwriting System: 1,Numeric,1 2 3 4 5 6 1111
+Automated Underwriting System: 2,Numeric,1 2 3 4 5 null 

```

Comment:
Please remove nulls from the value list for the same reason as blanks.

Reply:
Sure, this was done in the last commit.